### PR TITLE
LPS-166761: Add Elasticsearch Collapse support to the search API

### PR DESCRIPTION
**Pull request by peerkar**
- Story: https://liferay.atlassian.net/browse/LPS-166761
- Subtasks covered:
  - https://liferay.atlassian.net/browse/LPS-199864
  - https://liferay.atlassian.net/browse/LPS-199873

### CODE REVIEW

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/searcher/SearchRequestImpl.java [hunk 1]
```diff
@@ -50,6 +51,7 @@ public SearchRequestImpl(SearchRequestImpl searchRequestImpl) {
 		_aggregationsMap.putAll(searchRequestImpl._aggregationsMap);
 		_basicFacetSelection = searchRequestImpl._basicFacetSelection;
 		_complexQueryParts.addAll(searchRequestImpl._complexQueryParts);
+		_collapse = searchRequestImpl._collapse;
```
Comment (BryanEngler) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/searcher/SearchRequestImpl.java:54]: sort

**Comment by BryanEngler** [comment]
looks good to me, just the one small sorting sf

**Comment by BryanEngler** [comment]
running ci here https://github.com/liferay-search/liferay-portal/pull/1127

**Pull request closed without merging**